<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(21, 130, 213);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>261</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 26pt &quot;News706 BT&quot;;</string>
    </property>
    <property name="text">
     <string>   PRODUCTOS</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCodigo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Código:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Nombre:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>160</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Descripción:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCodigo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombre">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDescripcion">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>511</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Stock Mínimo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Stock Actual:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>250</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Precio de Venta:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Precio de Costo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>250</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Proveedor:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>250</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Almacén:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtStockMinimo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtStockActual">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPrecioCosto">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPrecioVenta">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>280</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cboProveedor">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>280</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <item>
     <property name="text">
      <string>Seleccionar Proveedor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Proveedor 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Proveedor 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Proveedor 3</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cboAlmacen">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>280</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <item>
     <property name="text">
      <string>Seleccionar Almacén</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Almacén 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Almacén 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Almacén 3</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="tblProductos">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>901</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <column>
     <property name="text">
      <string>Código</string>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nombre</string>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Descripción</string>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Stock Mínimo</string>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Stock Actual</string>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Precio en IGV</string>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Venta en IGV</string>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Proveedor</string>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Almacén</string>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnRegistrar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(255, 26, 10);</string>
    </property>
    <property name="text">
     <string>REGISTRAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../IMG/registrar.png</normaloff>../IMG/registrar.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnModificar">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>560</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(255, 26, 10);</string>
    </property>
    <property name="text">
     <string>MODIFICAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../IMG/modificar.png</normaloff>../IMG/modificar.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConsultar">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>560</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(255, 26, 10);</string>
    </property>
    <property name="text">
     <string>CONSULTAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../IMG/consultar.png</normaloff>../IMG/consultar.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEliminar">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>560</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(255, 26, 10);</string>
    </property>
    <property name="text">
     <string>ELIMINAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../IMG/eliminar.png</normaloff>../IMG/eliminar.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnListar">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>560</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;color: rgb(255, 255, 255);
background-color: rgb(255, 26, 10);</string>
    </property>
    <property name="text">
     <string>LISTAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../IMG/listar.png</normaloff>../IMG/listar.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>40</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../IMG/carrito de compra.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGrabar">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>560</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(255, 26, 10);</string>
    </property>
    <property name="text">
     <string>GRABAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../IMG/modificar.png</normaloff>../IMG/modificar.png</iconset>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>lblCodigo</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>txtCodigo</zorder>
   <zorder>txtNombre</zorder>
   <zorder>txtDescripcion</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>txtStockMinimo</zorder>
   <zorder>txtStockActual</zorder>
   <zorder>txtPrecioCosto</zorder>
   <zorder>txtPrecioVenta</zorder>
   <zorder>cboProveedor</zorder>
   <zorder>cboAlmacen</zorder>
   <zorder>tblProductos</zorder>
   <zorder>btnConsultar</zorder>
   <zorder>btnEliminar</zorder>
   <zorder>btnListar</zorder>
   <zorder>btnRegistrar</zorder>
   <zorder>label_11</zorder>
   <zorder>btnGrabar</zorder>
   <zorder>btnModificar</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>926</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
